clients/java/src/main/java/io/camunda/zeebe/client/impl/ZeebeClientBuilderImpl.java

```diff
@@ -83,6 +83,7 @@ public final class ZeebeClientBuilderImpl implements ZeebeClientBuilder, ZeebeCl
   public static final String USE_DEFAULT_RETRY_POLICY_VAR = "ZEEBE_CLIENT_USE_DEFAULT_RETRY_POLICY";
   private static final String TENANT_ID_LIST_SEPARATOR = ",";
   private static final boolean DEFAULT_PREFER_REST_OVER_GRPC = false;
+  public static final Duration DEFAULT_MESSAGE_TTL = Duration.ofHours(1);
```

Comment:
🙃 Public constants go first, according to codestyle 🙂

Reply:
need to check my intellij settings, don't we have this covered by auto-format config?

Reply:
Seems intelliJ only does this via "rearrange code", enabled it now, let's see how often it causes more changes than intended :D

Reply:
🤣